### PR TITLE
Linked editing doesn't work if variable is used as a parameter into a section

### PR DESCRIPTION
**Pull request by angelozerr**
Linked editing doesn't work if variable is used as a parameter into a section

Fixes https://github.com/redhat-developer/vscode-quarkus/issues/500

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

File: qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/services/definition/QuteDefinitionInForUserTagTest.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	public void it() throws Exception {
+		String template = "{#for item in items}\r\n" + //
+				"	{#linkItem it|em name=item.name item /}\r\n" + //
+				"{/for}";
+		testDefinitionFor(template, "test.qute", //
+				ll("test.qute", r(1, 12, 1, 16), r(0, 6, 0, 10)));
+	}
+
+	@Test
+	public void noIt() throws Exception {
```
Comment (fbricon) [inline qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/services/definition/QuteDefinitionInForUserTagTest.java:47]: what do it/noIt mean?

Comment (angelozerr) [reply qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/services/definition/QuteDefinitionInForUserTagTest.java:47]: See https://quarkus.io/guides/qute-reference#user_tags

Comment (angelozerr) [reply qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/services/definition/QuteDefinitionInForUserTagTest.java:47]: in user tag you can write parameter without value like

`{#user name}` 

in this case name is the it of the user tag.

You can hav ejust one it, so if you write

`{#user name name2}` 

name2 is not an it (it should have an error report for that)

**Comment by fbricon** [comment]
thanks @angelozerr

**Pull request merged**